x-pack/solutions/observability/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -284,7 +317,15 @@ export const generalSettings: RawSettingDefinition[] = [
       defaultMessage:
         'When recording, the agent instruments incoming HTTP requests, tracks errors, and collects and sends metrics. When set to non-recording, the agent works as a noop, not collecting data and not communicating with the APM Server except for polling for updated configuration. As this is a reversible switch, agent threads are not being killed when set to non-recording, but they will be mostly idle in this state, so the overhead should be negligible. You can use this setting to dynamically control whether Elastic APM is enabled or disabled.',
     }),
-    excludeAgents: ['nodejs', 'rum-js', 'js-base'],
+    excludeAgents: [
+      'nodejs',
+      'rum-js',
+      'js-base',
+      'opentelemetry/dotnet/elastic',
+      'opentelemetry/nodejs/elastic',
+      'opentelemetry/php/elastic',
+      'opentelemetry/python/elastic',
```

Comment:
@AlexanderWert did you intentionally left `opentelemetry/java/elastic` out of the exclude agents list?

Reply:
yes, it implements this